File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/C-C++/14.cpp
#include <iostream>
using namespace std;

/*14. Escreva um programa que solicite ao usuário dois números reais e ao final apresente na
tela o produto dos dois números informados da seguinte forma: "O produto dos numeros N
e X corresponde a Y"*/

int main(){
	float numero1, numero2;
	
	printf("Digite o primeiro numero real: ");
	scanf("%f", &numero1);
	
	printf("Digite o segundo numero real: ");
	scanf("%f", &numero2);
	
	printf("O produto dos numeros %0.2f e %0.2f corresponde a %0.2f", numero1, numero2, numero1 * numero2);
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 3/Python/14.py
"""
14. Elabore um algoritmo que solicite ao usuário 10 números reais e ao final apresente o
maior e o menor deles.
"""

i = 1
userInput = ""
numeros = []

while i <= 10:
    userInput = float(input("Digite um numero real: "))
    numeros.append(userInput)

    if(i > userInput):
        userInput = i

    i += 1

numeros.sort()
print("O numero maior é: ", numeros[-1])
print("O numero menor é: ", numeros[0])